### PR TITLE
fix: bump connect loader to beta without polyfill

### PR DESCRIPTION
**Pull request by cgdibble**
### Description

Use the new Connect Loader beta version that removes the polyfill from the downstream asset loader.

### Why are we making these changes? Include references to any related Jira tasks or GitHub Issues

Our build enforces flat deps and the polyfill is a dep required by old Hosted Fields.

### Groups who should review (if applicable)

❤️ Thank you!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -110,7 +110,7 @@
     "@krakenjs/zoid": "^10.3.1",
     "@paypal/common-components": "^1.0.35",
     "@paypal/funding-components": "^1.0.31",
-    "@paypal/connect-loader-component": "^1.1.0",
+    "@paypal/connect-loader-component": "1.1.0-beta.1",
```
Comment (siddy2181) [inline package.json:113]: We need to update `package_lock.json` too?

**Review by SancSalix — APPROVED** [review]
🌟

File: package.json (hunk 1 above)
Comment (cgdibble) [reply package.json:113]: @siddy2181 there isn't a lock file in checkout components.
We disable the lockfile generation here: https://github.com/paypal/paypal-checkout-components/blob/main/.npmrc#L3

**Review by siddy2181 — APPROVED** [review]
Let's ship it!!!!!

**Pull request merged**